src/modules/fancyzones/lib/FancyZonesDataTypes.h

```diff
@@ -32,6 +33,25 @@ namespace FancyZonesDataTypes
         Canvas
     };
 
+    struct DeviceIdData
+    {
+        std::wstring deviceName;
```

Comment:
The default constructor doesn't initialize all members, this impacts the function `empty()`

Reply:
`empty()` checks for `==` with `DeviceIdData{}`. That should initialize all. Could you clarify?

Reply:
You're right, `DeviceIdData{}` initializes everything, my bad.